src/components/lyricseditor/lyricseditor.js

```diff
@@ -0,0 +1,390 @@
+import escapeHtml from 'escape-html';
+
+import { LyricsApi } from '@jellyfin/sdk/lib/generated-client/api/lyrics-api';
+import { toApi } from 'utils/jellyfin-apiclient/compat';
+import dialogHelper from '../dialogHelper/dialogHelper';
+import layoutManager from '../layoutManager';
+import globalize from '../../scripts/globalize';
+import loading from '../loading/loading';
+import focusManager from '../focusManager';
+import dom from '../../scripts/dom';
+import '../../elements/emby-select/emby-select';
+import '../listview/listview.scss';
+import '../../elements/emby-button/paper-icon-button-light';
+import '../formdialog.scss';
+import 'material-design-icons-iconfont';
+import './lyricseditor.scss';
+import '../../elements/emby-button/emby-button';
+import '../../styles/flexstyles.scss';
+import ServerConnections from '../ServerConnections';
+import toast from '../toast/toast';
+import template from './lyricseditor.template.html';
+import templatePreview from './lyricspreview.template.html';
+import { deleteLyrics } from '../../scripts/deleteHelper';
+
+let currentItem;
+let hasChanges;
+
+function downloadRemoteLyrics(context, id) {
+    const api = toApi(ServerConnections.getApiClient(currentItem.ServerId));
+    const lyricsApi = new LyricsApi(api.configuration, undefined, api.axiosInstance);
```

Comment:
You should use the `getLyricsApi(api)` utility.

Reply:
I think that wasn't present or I couldn't find it when I initally coded this 🙃
Will update to use it.

Reply:
It probably hadn't been added yet 😅